commander/src/commands/transaction/sign.ts

```diff
@@ -88,7 +125,33 @@ export default class SignCommand extends BaseCommand {
 			...transactionObject,
 			networkIdentifier,
 		});
-		txInstance.sign(passphrase);
+
+		interface Keys {
+			readonly mandatoryKeys: Array<Readonly<string>>;
+			readonly optionalKeys: Array<Readonly<string>>;
+			readonly numberOfSignatures: number;
+		}
+
+		const keys = ({
+			mandatoryKeys,
+			optionalKeys,
+			numberOfSignatures,
+		} as unknown) as Keys;
```

Comment:
why do we need to type like this?

Reply:
because the parsed flags give undefined so, any ideas not to cast?

Reply:
flag is optional, so it could be undefined too unless it's used with required.
in that sense, it's better to check if it's undefined or not